web_src/less/_repository.less

```
@@ -86,6 +86,10 @@
         }
 
         .ui.list {
+            &.assignees .icon {
+                line-height: 1.8em;
```

```suggestion
                line-height: 2em;
```

I think it should match the height of the avatar which is also 2em.

---

Looks good